docs/en/connector-v2/sink/HdfsFile.md

```diff
@@ -244,32 +161,26 @@ HdfsFile {
     sink_columns = ["name","age"]
     is_enable_transaction = true
 }
-
 ```
 
-## Changelog
+### For kerberos simple config
 
-### 2.2.0-beta 2022-09-26
-
-- Add HDFS File Sink Connector
-
-### 2.3.0-beta 2022-10-20
-
-- [BugFix] Fix the bug of incorrect path in windows environment ([2980](https://github.com/apache/seatunnel/pull/2980))
-- [BugFix] Fix filesystem get error ([3117](https://github.com/apache/seatunnel/pull/3117))
-- [BugFix] Solved the bug of can not parse '\t' as delimiter from config file ([3083](https://github.com/apache/seatunnel/pull/3083))
-
-### 2.3.0 2022-12-30
-
-- [BugFix] Fixed the following bugs that failed to write data to files ([3258](https://github.com/apache/seatunnel/pull/3258))
-  - When field from upstream is null it will throw NullPointerException
-  - Sink columns mapping failed
-  - When restore writer from states getting transaction directly failed
+```
+HdfsFile {
+    fs.defaultFS = "hdfs://hadoopcluster"
+    path = "/tmp/hive/warehouse/test2"
+    kerberos_principal = "your_principal@EXAMPLE.COM"
+    kerberos_keytab_path = "/path/to/your/keytab/file.keytab"
+}
```

Comment:
Add hdfs_site_path , It is usually used in conjunction with hdfs_site_path.

Reply:
thanks. Just prepare to update and see that you have added.